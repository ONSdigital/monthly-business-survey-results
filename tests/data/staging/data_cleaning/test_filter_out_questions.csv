question,values
11,10000
42,1001
43,1002
46,1003
12,1004
47,1005
90,1006
40,1007
49,1008
146,1009
