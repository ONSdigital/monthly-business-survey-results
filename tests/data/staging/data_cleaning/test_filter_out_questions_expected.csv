question,values
42,1001
43,1002
46,1003
47,1005
90,1006
40,1007
49,1008
